### PR TITLE
package

### PR DESCRIPTION
**Pull request by vutuankiet4599**
## Related Tickets:

-   [#67866](https://edu-redmine.sun-asterisk.vn/issues/67866)

## WHAT this PR does?

- Cài đặt các package breeze, lang common, jquery

## Notes Impacted Areas:

-   N/A.

## Attachments:

image

### CODE REVIEW

**Comment by namnt0413** [comment]
oke

**Comment by vutuankiet4599** [comment]
ready

File: tests/Feature/Auth/PasswordConfirmationTest.php [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+
+    // public function testConfirmPasswordScreenCanBeRendered()
+    // {
+    //     $user = User::factory()->create();
+
+    //     $response = $this->actingAs($user)->get('/confirm-password');
+
+    //     $response->assertStatus(200);
+    // }
+
```
Comment (ghost) [inline tests/Feature/Auth/PasswordConfirmationTest.php]: ???

**Comment by vutuankiet4599** [comment]
ready

File: .sun-ci.yml [hunk 2]
```diff
@@ -59,8 +59,27 @@ jobs:
...
+          - vendor
+    workspace: shared
     before_script:
+      - cp .env.example .env.testing
       - composer install
+      - php artisan key:generate --env=testing
+      - php artisan config:cache
+      - php artisan config:clear
+      - php artisan cache:clear
+      - php artisan migrate --force
```
Comment (ghost) [inline .sun-ci.yml]:
```suggestion
      - php artisan migrate --force --env=testing
```

**Pull request merged**